### PR TITLE
#139- Renamed seed files

### LINKED ISSUE #139: Backend - Rename seed files
### Description

the seed file names (src/backend/src/prisma/seed-data) are too simple. For example, 'projects.ts' does not explain that it is a seed file, and this can make things confusing when searching for files. They should be renamed to follow how the controller/routes files are (e.g., projects.seed.ts).

### Background

.

### Proposed Solution

change the seed file names in `src/backend/src/prisma/seed-data` to include the `.seed`

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by ryanhowe28**
## Changes
Re-named seed-data files
Added "seed" to the name in all the files in backend/src/prisma/seed-data

Closes #139

### CODE REVIEW

**Review by Ari-S-123 — APPROVED** [review]
lgtm

**Review by Ari-S-123 — CHANGES_REQUESTED** [review]
You need to update the imports in seed.ts too.

File: src/backend/src/prisma/seed.ts [hunk 1]
```diff
@@ -40,17 +40,6 @@ const performSeed: () => Promise<void> = async () => {
     });
   }
 
-  for (const seedTeam of dbSeedAllTeams) {
```
Comment (anthonybernardi) [inline src/backend/src/prisma/seed.ts:43]: this was deleted by accident

**Review by anthonybernardi — CHANGES_REQUESTED** [review]

**Review by anthonybernardi — APPROVED** [review]
nice work!

**Pull request merged**